otx/algorithms/segmentation/configs/configuration.yaml

```diff
@@ -149,7 +149,7 @@ learning_parameters:
     warning: null
   enable_early_stopping:
     affects_outcome_of: TRAINING
-    default_value: true
+    default_value: false
```

Comment:
I don't think we need to disable this.. I hope you increase patience for early stop as workaround.

Reply:
Yes sorry, it was just for debugging. I forgot to revert it back